tests/test_items.cpp

```diff
@@ -983,6 +983,57 @@ TEST_F(ItemTest, CreateItemsWithEmptyVector)
     }
     FAIL();
 }
+
+TEST_F(ItemTest, SendItemsWithVector)
+{
+    std::vector<ews::message> messages;
+    auto message = ews::message();
+    std::vector<ews::mailbox> recipients;
+    for (int i = 0; i < 10; i++)
+    {
+        message.set_subject("?");
+        recipients.push_back(ews::mailbox("whoishere@home.com"));
+        message.set_to_recipients(recipients);
+        messages.push_back(message);
+    }
+
+    auto message_ids =
+        service().create_item(messages, ews::message_disposition::save_only);
+    ews::internal::on_scope_exit remove_messages([&] {
+        for (auto& id : message_ids)
+        {
+            service().delete_item(id);
+        }
+    });
+
+    EXPECT_NO_THROW(service().send_item(message_ids));
+}
+
+TEST_F(ItemTest, SendItemsWithEmptyVector)
+{
+    std::vector<ews::message> messages;
+    auto message = ews::message();
+    std::vector<ews::mailbox> recipients;
+    for (int i = 0; i < 10; i++)
+    {
+        message.set_subject("?");
+        recipients.push_back(ews::mailbox("whoishere@home.com"));
+        message.set_to_recipients(recipients);
+        messages.push_back(message);
+    }
+
+    auto message_ids =
+        service().create_item(messages, ews::message_disposition::save_only);
+    ews::internal::on_scope_exit remove_messages([&] {
+        for (auto& id : message_ids)
+        {
+            service().delete_item(id);
+        }
+    });
```

Comment:
Why do you create messages here when all you do is pass an empty vector to send_item?

Reply:
I see what you mean, i could have just as well used a `std::vector<item_id>` instead. I'll change that.